hooks/environment

```diff
@@ -78,7 +78,6 @@ if [[ -n "$s3_bucket" ]] ; then
       set +o allexport
     elif [[ $? -eq 2 ]] ; then
       echo "+++ :warning: Failed to check if $key exists" >&2;
```

Comment:
@chloeruka is this the line you meant to remove the warning from? The other one is the ssh_private_key check and that's been there for years

Reply:
Yep, sorry, been meaning to come back to this as it was a hasty change applied in the middle of a support catchup, I'll revert